package.json

```diff
@@ -18,12 +18,14 @@
     "dist/*"
   ],
   "main": "dist/_hyperscript.min.js",
+  "types": "dist/_hyperscript.d.ts",
   "bin": {
     "_hyperscript": "src/bin/node-hyperscript.js"
   },
   "scripts": {
     "test": "mocha-chrome test/index.html",
-    "dist": "cp -r src/* dist/ && npm run-script terser && gzip -k -f dist/_hyperscript.min.js > dist/_hyperscript.min.js.gz && exit",
+    "dist": "cp -r src/* dist/ && npm run-script terser && npm run-script typings && gzip -k -f dist/_hyperscript.min.js > dist/_hyperscript.min.js.gz && exit",
+    "typings": "npx --yes tsc --declaration dist/_hyperscript.js --allowJs --emitDeclarationOnly --skipLibCheck",
```

Comment:
probably  a good idea is to add the argument ` --declarationMap` to generate maps for typing, so later within IDE when you click on - go to definition - you jump directly on source file instead of types